build/generateIconsDoc.php

```diff
@@ -26,6 +26,12 @@
 use Leafo\ScssPhp\Compiler;
 use Leafo\ScssPhp\Formatter\Crunched;
 
+// You need svgexport to run this script
+if (!`which svgexport`) {
```

Comment:
🙈 please do something else, but not this inside backticks. Something like shell_exec at least